aspnet/tutorials/nano-server.rst

```diff
@@ -86,8 +86,11 @@ Run the following commands in the PowerShell session that was created earlier:
 
   Install-PackageProvider NanoServerPackage
   Import-PackageProvider NanoServerPackage
+  Install-NanoServerPackage -Name Microsoft-NanoServer-Storage-Package
```

Comment:
I'm okay with publishing this workaround. I just want the Note below to explicitly call out that this is a workaround and will not be required in the future